### ISSUE TITLE
Fungerer extends med nytt pakkenavn? 

### ISSUE THREAD

**Issue by lostfields** [tfso/eslint-config-tfso#5]
Med endring på pakkenavn, vil gamle pakkereferanse i ESlint config fungere da? 

https://github.com/tfso/eslint-config-tfso/blob/7eb065b8c47970fefe067fd1db7ebe98137c384e/package.json#L2

Kort referanse som fungerer fordi pakkenavn var `eslint-config-tfso`

```
{
  "extends": [
    "tfso/typescript"
  ]
}
```

**Comment by OysteinVesth** [comment]
Good catch @lostfields, var faktisk ikke klar over at pakka var public, så var unødvendig av meg uansett. Bytter tilbake navn og fjerner deployment av pakke, men beholder tags.

**Comment by OysteinVesth** [comment]
Renamet til gammelt navn, men beholder changelog GH action

**Comment by lostfields** [comment]
Det vil fungere iht dokumentasjonen på https://eslint.org/docs/developer-guide/shareable-configs#npm-scoped-modules men man må endre extend fra `tfso` til `@tfso`. "Problemet" kommer når man skal spesifisere en spesiell config i prosjektet, f.eks. `tfso/react`, og med scoped navn så må man bruke `@tfso/eslint-config/react` istedenfor.

Selv om vi endrer navn på package.json så vil vel det gamle fungere, for det er jo navnet på pakkareferansen som bestemmer navnet på mappen under node_modules.

Bare kjøre på slik det var @OysteinVesth ?
